src/Test/Perf/Utilities/RelativeDirectory.cs

```diff
@@ -9,44 +9,66 @@ namespace Roslyn.Test.Performance.Utilities
 {
     public class RelativeDirectory
     {
-        string _workingDir;
+        private string _workingDir;
 
-        public RelativeDirectory([CallerFilePath] string workingFile = "")
+
+        public RelativeDirectory()
         {
-            _workingDir = Directory.GetParent(workingFile).FullName;
         }
 
-        public string MyWorkingDirectory => _workingDir;
+        public RelativeDirectory(string workingDir)
```

Comment:
Should this be settable both in the ctor and ```SetWorkingDirectory```?  I would suggest one or the other.